src/main/groovy/com/palantir/gradle/docker/PalantirDockerPlugin.groovy

```diff
@@ -89,6 +89,9 @@ class PalantirDockerPlugin implements Plugin<Project> {
                     buildCommandLine.addAll('--build-arg', "${buildArg.getKey()}=${buildArg.getValue()}")
                 }
             }
+            if (ext.pull) {
```

Comment:
if wonder if we should instead allow for a more general mechanism that can also specify other cmd line flags, etc `--quiet`, etc?